core/renderers/geras/info.js

```diff
@@ -376,6 +377,67 @@ Blockly.geras.RenderInfo.prototype.getElemCenterline_ = function(row, elem) {
   return result;
 };
 
+/**
+ * @override
+ */
+Blockly.geras.RenderInfo.prototype.alignRowElements_ = function() {
+  if (!this.isInline) {
+    Blockly.geras.RenderInfo.superClass_.alignRowElements_.call(this);
+    return;
+  }
+
+  // Walk up each row from the bottom, keeping track of the right input edge.
```

Comment:
Walk backwards through elements in a row, ...

Reply:
Maybe "Walk backgrounds through rows on the block"? Because we're not going through the elements in the row.